### PR TITLE
help

### PR DESCRIPTION
**Pull request by edgarlokys**
https://edgarlokys.github.io/layout_catalog/
https://edgarlokys.github.io/layout_catalog/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,8 +5,386 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;1,700&display=swap" rel="stylesheet">
```
Comment (VitaliyBondarenko1982) [inline src/index.html:10]: Looks like incorrect link
```suggestion
 <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
```

File: src/index.html [hunk 2]
```diff
@@ -5,8 +5,386 @@
...
+          <li class="nav__item">
+            <a href="#video" class="nav__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="cataloge">
```
Comment (VitaliyBondarenko1982) [inline src/index.html:67]:
```suggestion
      <div class="catalog">
```

File: src/styles/blocks/cataloge.scss [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+      margin-right: 0;
+    }
+  
+    &:nth-last-child(-n + 4) {
+      margin-bottom: 0;
+    }
+  }
+}
+
+.card {
```
Comment (VitaliyBondarenko1982) [inline src/styles/blocks/cataloge.scss:23]: Move styles fo card in separate file

File: src/styles/blocks/cataloge.scss [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+    font-family: "Roboto", sans-serif;
+    font-size: $title-size;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-accent-color;
+    margin-bottom: 4px;
+  }
+  
+  &__code {
+    font-family: "Roboto-Regular", sans-serif;
```
Comment (VitaliyBondarenko1982) [inline src/styles/blocks/cataloge.scss:50]: Where did it come from font-family Roboto-Regular?
```suggestion
    font-family: "Roboto", sans-serif;
```

File: src/styles/blocks/cataloge.scss [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+  &__price-count {
+    font-family: "Roboto", sans-serif;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $main-accent-color;
+  }
+  
+  &__button {
+    font-family: "Roboto-Bold", sans-serif;
```
Comment (VitaliyBondarenko1982) [inline src/styles/blocks/cataloge.scss:100]: Where did it come from font-family Roboto-Regular?
```suggestion
    font-family: "Roboto", sans-serif;
```

File: src/styles/blocks/cataloge.scss [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+    border: 1px solid rgba(255, 255, 255, 0);
+    transition: 0.3s;
+  }
+  
+  &__button:hover {
+    border: 1px solid $main-blue-accent;
+    background-color: #fff;
+    color: $main-blue-accent;
+    transition: 0.3s;
+  }
```
Comment (VitaliyBondarenko1982) [inline src/styles/blocks/cataloge.scss:119]: use scss syntax. heck other places
```suggestion
    &:hover {
      border: 1px solid $main-blue-accent;
      background-color: #fff;
      color: $main-blue-accent;
      transition: 0.3s;
    }
  }
```

File: src/styles/blocks/cataloge.scss [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-align: center;
+    padding: 11px 0;
+    background-color: $main-blue-accent;
+    border-radius: 5px;
+    color: #fff;
+    text-transform: uppercase;
+    border: 1px solid rgba(255, 255, 255, 0);
```
Comment (VitaliyBondarenko1982) [inline src/styles/blocks/cataloge.scss:110]:
```suggestion
    border: 1px solid #fff;
```

Comment (VitaliyBondarenko1982) [reply src/styles/blocks/cataloge.scss:110]: Move all colors to variables

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]

**Review by VitaliyBondarenko1982 — APPROVED** [review]

**Review by mykhalenych — APPROVED** [review]

**Pull request closed without merging**